### DISCUSSION TITLE
Methods to Publish online Built Binary Files

### DISCUSSION THREAD

**Discussion by caibinwav123456** [community/community#29997] [Apps, API and Webhooks]
Discuss various ways to publish or share products built from your Github repositories.
Any actions & APIs that can help with this?

**Comment by jfullstackdev** [comment]
you mean releases ? 

GitHub has its own Releases tab

you may want to check my own repo as an example, 

https://github.com/jdevstatic/lightweight-web-server

look how I utilize it and the automated report of how many commits from a specific point
